manifests/t/Texmaker/Texmaker/5.1.3/Texmaker.Texmaker.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Texmaker.Texmaker
+PackageVersion: 5.1.3

```

```suggestion
PackageVersion: 5.1.3.0
```
This must match Control Panel.